config/environments/production.rb

```diff
@@ -84,4 +84,9 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.session_store = :redis_store,
+    servers: [ENV.fetch("REDIS_URL")],
```

Comment:
I think we need something like `.env` to list required environment variables.

Reply:
+1

Reply:
cfe3102